emmo/README.md

```diff
@@ -0,0 +1,102 @@
+
+8. `https://w3id.org/emmo/{DOMAIN}/ --> https://raw.githubusercontent.com/emmo-repo/{REPO_NAME}/master/{DOMAIN}.ttl`
+   - `{DOMAIN}.ttl` file in the root of the master branch.
+
+9. `https://w3id.org/emmo/{DOMAIN}/{VERSION} --> https://raw.githubusercontent.com/emmo-repo/{REPO_NAME}/{VERSION}/{DOMAIN}{.html|.ttl}`
+   - If the user is accessing this from a browser, redirect to html documentation for given version on GitHub Pages.
+   - Otherwise, redirect to squashed .ttl file for given version on GitHub Pages.
+   - Special case for inferred ontology `https://w3id.org/{DOMAIN}/{VERSION}/{DOMAIN}-inferred --> https://emmo-repo.github.io/{REPO_NAME}/versions/{VERSION}/{DOMAIN}-inferred.ttl`
+
+10. `https://w3id.org/emmo/{DOMAIN}/{VERSION}/ --> https://raw.githubusercontent.com/emmo-repo/{REPO_NAME}/{VERSION}/{DOMAIN}.ttl`
+   - `{DOMAIN}.ttl` file in the root of GitHub branch for the given version.
+
+11. `https://w3id.org/emmo/{DOMAIN}/{PATH}/{MODULE} --> https://raw.githubusercontent.com/emmo-repo/{REPO_NAME}/master/{PATH}/{MODULE}.ttl`
+    - `{PATH}/{MODULE}.ttl` file in master branch.
+
+12. `https://w3id.org/emmo/{DOMAIN}/{VERSION}/{PATH}/{MODULE} --> https://raw.githubusercontent.com/emmo-repo/{REPO_NAME}/{VERSION}/{PATH}/{MODULE}.ttl`
+    - `{PATH}/{MODULE}.ttl` file for given version and module.
+
+
+### Meaning of placeholders
+- `{NAME}`: Name part of an IRI, i.e. what follows after the hash sign.
+- `{VERSION}`; Version number. Must start with a digit to distinguish it from domain or path names.
+- `{PATH}`: Directory path within a github repository
+- `{MODULE}`: Filename of turtle file with the final .ttl stripped off
+- `{DOMAIN}`: Name of domain ontology. Initial `domain-` is stripped off.
+- `{REPO_NAME}`: Name of GitHub repository for domain ontology.
+
+
+## Expected organisation of repositories for domain ontologies
+
+### GitHub repository
+In the root of the repository we should have a file called {DOMAIN}.ttl. This should be the main file for the ontology. It should exist to make the ontology FAIR.
+
+### GitHub Pages
+Using GitHub pages is optional.
+There are several use cases for using cases for using GitHub Pages:
+- The main turtle file is generated from another semantic source.
+  In this case it make sense to distribute the ontology on GitHub pages to avoid polluting your GitHub repository.
+- Your ontology consists of many modules importing each other.
+  Loading them recursively from GitHub may be very slow.
+  To speed up the loading process, you could merge (squash) all modules into a single turtle file, which you could store on GitHub Pages.
+- You want to publish the inferred ontology, so that the user have full to the full inferred ontology without having to first running a reasoner.
+- Reference html documenting of classes and relations in the ontology.
+  The main aim of this document is to making the IRIs resolvable in a browser as human readable documentation of the concepts.
```

Comment:
```suggestion
- _The main turtle file is generated from another semantic source._
  In this case it make sense to distribute the ontology on GitHub pages to avoid polluting your GitHub repository.
- _Your ontology consists of many modules importing each other._
  Loading them recursively from GitHub may be very slow.
  To speed up the loading process, you could merge (squash) all modules into a single turtle file, which you could store on GitHub Pages.
- _You want to publish the inferred ontology, so that the user have full to the full inferred ontology without having to first running a reasoner._
- _Reference html documenting of classes and relations in the ontology._
  The main aim of this document is to making the IRIs resolvable in a browser as human readable documentation of the concepts.
```